src/ui/flutter_app/lib/l10n/intl_es.arb

```diff
@@ -917,7 +917,7 @@
     "@atEnd": {
         "description": "At the end of the move list."
     },
-    "tapBackAgainToLeave": "Vuelve a tocar para salir.",
+    "tapBackAgainToLeave": "Vuelva a tocar para salir.",
```

Comment:
Can you help sync to line 154 of resources.dart if that's convenient?

Reply:
I'm sorry, but I don't know about you mean.

Reply:
Debido al mecanismo operativo de la aplicación Flutter, es necesario modificar sincrónicamente el código correspondiente a la línea 154 de resource.dart en este directorio de archivos para que surta efecto en el programa. Puede abrir el archivo y ver el contenido de esa línea para saber que hay un carácter La copia de la cadena está ahí.

Reply:
Done. See https://github.com/calcitem/Sanmill/commit/017becaeb125ed4dcf09338b947acf4a453f3c5f